group.go

```diff
@@ -9,8 +9,9 @@ import (
 // When one actor (function) returns, all actors are interrupted.
 // The zero value of a Group is useful.
 type group struct {
-	actors       []actor
-	closeTimeout time.Duration
+	actors          []actor
+	closeTimeout    time.Duration
+	orderedShutdown bool
```

Comment:
nit: this seems to behave as a sync shutdown rather than ordering. in line 24, am i understanding correctly that this is shutting down as a queue rather than a stack? wouldn't it make sense to have the shutdown behaviour be reversed since that's often how dependencies would be defined

Reply:
I agree I should rename to `sync`. We do need the queue semantics though. We start Redis last and need it to be shutdown last.

Reply:
That seems kind of counter intuitive to me. If a service is started last, it would be expected that it's not a dependency of the previous ones generally since the other services is expected to not be able to run without it. Are we actually dependent on redis being started last here? It seems like we'd probably want to change the runnable order in our orchestrator code?